.node-version

```diff
@@ -0,0 +1 @@
+v20.11.1
```

Comment:
The README says to use Node `18` or higher – does it make sense to require version `20` here (and in `.nvmrc`)?

Reply:
I put `20` as it's the current LTS version, but personally had issues running with `16` that went away once I moved to 18 or 20...
So this is more of a suggested (`20`) vs required (`18`) 🤔

Reply:
Ah okay sure, fine with me 🙂